packages/big-bang-core/bb.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+wget -q "https://umbrella-bigbang-releases.s3-us-gov-west-1.amazonaws.com/umbrella/${1}/package-images.yaml" -O "images.yaml"
```

Comment:
this entire bash script is terrifying. Why aren't we using the images yaml from releases?

Reply:
removed and changed to pull images from release object at runtime